core-bundle/src/Resources/contao/library/Contao/Controller.php

```diff
@@ -1762,7 +1767,7 @@ public static function addImageToTemplate($objTemplate, $arrItem, $intMaxWidth=n
 						}
 						catch (\Exception $e)
 						{
-							$objTemplate->$strHrefKey = static::addFilesUrlTo(System::urlEncode($arrItem['imageUrl']));
+							$objTemplate->$strHrefKey = static::addFilesUrlTo($arrItem['imageUrl']);
```

Comment:
Shouldn't this be `$imageUrl` instead of `$arrItem['imageUrl']`?

Reply:
Changed in fe3df76